docs/reqs/reqs.tex

```diff
@@ -232,6 +238,25 @@ \section{Non-Functional Requirements}\label{section:nonfuncrec}
 
 TODO: We might have to split non-functional requirements into nf-reqs and constraints (the capstone docs say they're separate, but PC's don't).
 
+% Camera and ML points
+\subsection{Camera}
+\begin{enumerate}[{NF.CAM}.1]
+	\item The cameral must use a standard interface, for future replacability
```

Comment:
Which standard interface? If not specified then this sounds more like a functional requirement